lib/definition/model.rb

```diff
@@ -58,6 +58,14 @@ def ==(other)
       @_attributes == other.instance_variable_get(:@_attributes)
     end
 
+    def eql?(other)
+      self == other
+    end
+
+    def hash
```

Comment:
in case you are wondering what this is, see https://rubyapi.org/3.1/o/object#method-i-hash